WorkoutGeneratorApp/android/.project

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should this file be checked in?

Reply:
Idk, it was not in the .gitignore that came with the project and people online said the build projects should be committed so...